storage/remote/googleiam/README.md

```diff
@@ -0,0 +1,5 @@
+googleiam package
```

Comment:
If you want readme for package, probably some main file in the package or `doc.go` would be better? 🤔

Reply:
This is the same as azure https://github.com/prometheus/prometheus/blob/main/storage/remote/azuread/README.md

I followed the same same style for consistency. Let me know if you want me to change it.

Reply:
Yea, it's weird. Weird formatting and only one sentence. I would just kill this file and put that in a single Go file we have as a package comment (on `googleiam.go`) https://go.dev/blog/godoc#:~:text=Comments%20on%20package%20declarations%20should%20provide%20general%20package%20documentation